tests/console/system_prepare.pm

```diff
@@ -22,6 +22,7 @@ use version_utils 'is_sle';
 use serial_terminal 'add_serial_console';
 use bootloader_setup qw(change_grub_config grub_mkconfig);
 use registration;
+use services::registered_addons;
```

Comment:
As we use only `full_registered_check` from that import, we could limit it to that method only:
```perl
use services::registered_addons 'full_registered_check';
```